modules/liveyieldAnalyticsAdapter.js

```diff
@@ -0,0 +1,200 @@
+import adapter from 'src/AnalyticsAdapter';
+import adaptermanager from 'src/adaptermanager';
+import CONSTANTS from 'src/constants.json';
+import * as utils from 'src/utils';
+
+const {
+  EVENTS: { BID_REQUESTED, BID_TIMEOUT, BID_RESPONSE, BID_WON }
+} = CONSTANTS;
+
+const prebidVersion = '$prebid.version$';
+
+const adapterConfig = {
+  /** Name of the `rta` function, override only when instructed. */
+  rtaFunctionName: 'rta',
+
+  /** This is optional but highly recommended. The value returned by the
+   *  function will be used as ad impression ad unit attribute value.
+   *
+   *  As such if you have placement (10293845) or ad unit codes
+   *  (div-gpt-ad-124984-0) but you want these to be translated to meaningful
+   *  values like 'SIDEBAR-AD-01-MOBILE' then this function shall express this
+   *  mapping.
+   */
+  getAdUnitName: function(placementOrAdUnitCode) {
+    return placementOrAdUnitCode;
```

Comment:
In that case - it may ideal to either surround the function calls in a `try/catch` block or validate the overridden functions set in the `config.options` to handle any accidental setups.